tools/report-converter/tests/unit/sparse_output_test_files/sparse1.out

```diff
@@ -0,0 +1,3 @@
+kernel/trace/trace.h:1499:38: warning: incorrect type in argument 1 (different address spaces)
```

Comment:
It would be better to add a simple C file under this directory which contains multiple bugs. This way we could reproduce the same output easily.

For example the first bug would be a single line bug similar to this:
```
main.c:1:5: warning: message1
```

This way we would test that we are able to parse single lines.

The second bug would be a bug which contains details:
```
main.c:2:4: warning: message2
main.c:2:4:    details1
```

The third bug would be similar to the second bug.

So the whole output would look like this:
```
main.c:1:5: warning: message1
main.c:2:4: warning: message2
main.c:2:4:    details1
main.c:4:7: warning: message3
main.c:4:7:    details2
main.c:4:7:    details3
```
Please check the test folder for eslint: https://github.com/Ericsson/codechecker/tree/master/tools/report-converter/tests/unit/eslint_output_test_files